pypeit/core/flux_calib.py

```diff
@@ -1221,22 +1228,18 @@ def standard_zeropoint(wave, Nlam, Nlam_ivar, Nlam_gpm, flam_true, mask_balm=Non
     zeropoint ( `numpy.ndarray`_):
       Spectroscopic zeropoint.
     """
-    if np.any(np.invert(np.isfinite(Nlam_ivar))):
+    if np.any(np.logical_not(np.isfinite(Nlam_ivar))):
         msgs.warn("NaN are present in the inverse variance")
+    ivar_bpm = np.logical_not(np.isfinite(Nlam_ivar) & (Nlam_ivar > 0))
```

Comment:
It seems you should be using the Nlam_gpm here as well? Also, consider making ivar_bpm a gpm since the entire routine uses gpms and having both can be confusing.

Reply:
Okay I'm fine with tis.